packages/@jsii/integ-test/test/build-cdk.test.ts

```diff
@@ -0,0 +1,96 @@
+import { mkdtemp, remove } from 'fs-extra';
+import * as path from 'path';
+import * as Octokit from '@octokit/rest';
+import { downloadReleaseAsset, minutes, ProcessManager, extractFileStream } from '../utils';
+import * as dotenv from 'dotenv';
```

Comment:
Make sure to order the imports:
- external, alphanumerically sorted (`@octokit/rest`, `doting`, `fs-extra`, `path`)
- internal (`../utils`)